File: Python/Programmers/예산.py
def solution(d, budget):
    answer = 0

    d.sort()
    # print(d)
    tmp = 0
    # cnt = 0
    for i in d:
        tmp += i
        if budget >= tmp:
            # print(tmp)
            # tmp += i
            answer += 1

    return answer

File: Java/Programmers/예산.java
import java.util.*;

class Solution {
    public int solution(int[] d, int budget) {
        int answer = 0;

        Arrays.sort(d);
        // System.out.println(d);

        int tmp = 0;
        for (int i: d) {
            tmp += i;
            if (budget >= tmp) {
                answer += 1;
            }
        }

        return answer;
    }
}